packages/cashier/src/components/cashier-onboarding/cashier-onboarding.tsx

```diff
@@ -47,6 +47,7 @@ type TCashierOnboardingProps = {
 };
 
 const CashierOnboarding = ({
+    // eslint-disable-next-line react/prop-types
```

Comment:
This is fixed in @vinu-deriv's https://github.com/binary-com/deriv-app/pull/6513.